aggregator/settings/base.py

```diff
@@ -130,36 +130,25 @@
 
 sentry_dsn = os.environ.get("SENTRY_DSN", None)
 if sentry_dsn:
-    import raven  # noqa
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
 
-    RAVEN_CONFIG = {"dsn": sentry_dsn, "environment": os.environ.get("ENV", None)}
-    INSTALLED_APPS.append("raven.contrib.django.raven_compat")
+    sentry_sdk.init(
+        dsn="https://examplePublicKey@o0.ingest.sentry.io/0",
+        integrations=[DjangoIntegration()],
+        traces_sample_rate=1.0,
+        send_default_pii=False,
+    )
 
     LOGGING = {
         "version": 1,
         "disable_existing_loggers": False,
         "filters": {
             "require_debug_false": {"()": "django.utils.log.RequireDebugFalse"}
         },
-        "handlers": {
-            "sentry": {
-                "level": "ERROR",
-                "class": "raven.contrib.django.raven_compat.handlers.SentryHandler",
-            },
-            "null": {"class": "logging.NullHandler"},
-        },
         "loggers": {
             # Silence DisallowedHost exception by setting null error handler
-            "django.security.DisallowedHost": {
-                "handlers": ["null"],
-                "propagate": False,
-            },
-            # Send middleware errors to sentry
-            "api.middleware": {
```

Comment:
I can't test this for you, but I'm pretty sure dropping this handler basically just disables logging to sentry.

I think you need to replace
https://github.com/DemocracyClub/aggregator-api/blob/c63d45ffa455dae725c00d223e6a7ce56fccf651/aggregator/apps/api/middleware.py#L22
with a call to `capture_message` or `capture_exception` so that you aren't just logging exceptions to nowhere.

This middleware is there the API always returns JSON even when there is an exception (it prevents that annoying thing where your clients are expecting JSON but then Django throws errors as HTML :roll_eyes: ), but this means this app never bubbles an exception all the way to the top so a default sentry config won't ever catch anything to log.